docs/sources/next/extensions/explore.md

```diff
@@ -51,10 +47,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-cache</h4>
         <p>Enable vendoring remote HTTP modules to a single source-control-friendly file</p>
     </a>
-    <a href="https://github.com/fornfrey/xk6-celery" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements.
https://github.com/fornfrey/xk6-celery/issues/7
https://github.com/fornfrey/xk6-celery/issues/6